lmod_proxy/edx_grades/actions.py

```diff
@@ -30,6 +30,16 @@ def post_grades(gradebook, form):
     csv_file.seek(0)
     results = None
     try:
+        # Create assignment to explicitly set max points.
```

Comment:
Don't we need to do this for every assignment, not just the first one? Each assignment may have its own max points